apps/server/src/modules/board/service/board-do-copy-service/recursive-copy.visitor.ts

```diff
@@ -123,6 +124,25 @@ export class RecursiveCopyVisitor implements BoardCompositeVisitorAsync {
 		this.copyMap.set(original.id, copy);
 	}
 
+	async visitDrawingElementAsync(original: DrawingElement): Promise<void> {
+		const copy = new DrawingElement({
```

Comment:
Is there anything that needs to be done to ensure there is an actual new drawing on the tldraw side?
Are there any edge cases, eg when the drawing is copied within the same board?

Reply:
With deleting drawingName and using simply id instead now there should be no problem with copying - we will just generate new id and pass it to tldraw.